lib/ExpandableListItem/ExpandableListItem.js

```diff
@@ -173,7 +173,7 @@ module.exports = kind(
 	components: [
 		// headerContainer required to avoid bad scrollWidth returned in RTL for certain text
 		// widths (webkit bug)
-		{name: 'headerContainer', kind: Control, classes: 'moon-expandable-list-item-header moon-expandable-picker-header moon-expandable-list-header', onSpotlightFocus: 'headerFocus', ontap: 'expandContract', components: [
+		{name: 'headerContainer', classes: 'moon-expandable-list-item-header moon-expandable-picker-header moon-expandable-list-header', onSpotlightFocus: 'headerFocus', ontap: 'expandContract', components: [
```

Comment:
Alternatively, I wonder if we can just explicitly declare `kind: Item` instead as it seems the intention of the `defaultKind` is to be the default kind of the child controls added to the client control. For example, if `ExpandablePicker` had been engineered a little differently and didn't completely overwrite the components of `ExpandableListItem` (maybe some future work to unify all of these variants), with its setting of the `defaultKind` to `CheckboxItem`, that would cause problems for the header here.